CRM/Event/Page/ParticipantListing.php

```diff
@@ -75,7 +75,7 @@ public function run() {
       ) . '.php';
     $error = include_once $classFile;
     if ($error == FALSE) {
-      CRM_Core_Error::statusBounce('Participant listing code file: ' . $classFile . ' does not exist. Please verify your custom particpant listing settings in CiviCRM administrative panel.');
+      CRM_Core_Error::statusBounce(ts('Participant listing code file: %1 does not exist. Please verify your custom particpant listing settings in CiviCRM administrative panel.', [1 => $classFile]));
```

Comment:
I didn't even know this existed! It's a bit technical and seems unlikely to be something a user would deal with - so it's a judgement call I guess whether to include it in translation.

Reply:
Also participant is spelled wrong. (particpant)

Reply:
Good points - I think I've managed to add a spelling fix - the other is a judgement call as you say & I'm on the side of 'getting this merged'